### ISSUE TITLE
[GCP] Add missing metrics using CLI

### ISSUE THREAD

**Issue by geostant** [elastic/beats#25065]
**Describe the enhancement:**

As a cloud-infra team we would like to separate our ways from the cloud provider's monitoring and alerting in-house tools and start using metricbeats and stack for monitoring and alerting.

While researching what is available in the modules I found missing metrics that I would like to be added via the CLI.

**Describe a specific use case for the enhancement or feature:**

- GCP Autoscaler max limit about to be reached 
  calculation when `(capacity/maxNumReplicas)*100 >= 90%`
  - CLI: instance_group.managed.maxNumReplicas (the upper limit of max instances in a group)
  - API: autoscaler.capacity (in service count of instances)
  
- GCP Autoscaler failed to spin new capacity
  calculation when `capacity != targetSize after 15 minutes` (they should match once capacity is up and running) 
  - CLI: instance_group.managed.targetSize (the count of instances that we desire to run)
  - API: autoscaler.capacity

So to sum up, I'm missing for GCP CLI the following
- instance_group.managed.maxNumReplicas
- instance_group.managed.targetSize

To grab with the CLI the instance_group data you can run the following:
`gcloud compute instance-groups managed describe --zone $ZONE $INSTANCE_GROUP_NAME`

Rel: https://github.com/elastic/cloud/issues/75183
